### PR TITLE
Add gamecodes for AKB48 and Synchronica, Banapass HW info

### PR DESCRIPTION
**Pull request by EmiMidnight**

### CODE REVIEW

File: namco/io.md [hunk 1]
```diff
@@ -40,6 +40,22 @@ External serial interface. Connects to motherboard serial, and exposes JVS and s
   - (internal) 1x 7pin (to power supply 12V and 5V, pinout left-right: GND GND NC GND 12V 5V)
   - (internal) 1x 4pin (to JVS expansion card, pinout left-right: V+ D+ D- V-)
 
+## H055 CARD-IF
+
+Banapass card reader. Connects to the motherboard via a custom serial port connector.
```
Comment (Shizmob) [inline namco/io.md:45]: Oh, does this hook up to the SERIAL3 (CARD IF)?

File: namco/io.md [hunk 2]
```diff
@@ -40,6 +40,22 @@ External serial interface. Connects to motherboard serial, and exposes JVS and s
...
+* Label: H055 CARD-IF PCB
+* Copyright: 2010 NAMCO BANDAI Games Inc.
+* Chips:
+  - RENESAS F3062F25V (QFP100 Micro-controller)
+  - Toshiba TD62083AFG (8ch Darlington Sink Driver for the LEDs)
+  - Sony RC-S620/S (NFC Reader Module via UART)
+  - Murata Piezo Buzzer
+* Connectors:
+  - (external) 1x 3pin (CN2)
+  - (external) 1x 9pin (CN1, SERIAL + POWER, only 7 pins are actually connected on the header)
```
Comment (Shizmob) [inline namco/io.md:57]: Are you sure the remaining pins aren't power and GND? I've made that mistake before, as 12V is typically routed using a pretty wide trace.

File: namco/io.md (hunk 1 above)
Comment (EmiMidnight) [reply namco/io.md:45]: It does!

File: namco/io.md (hunk 2 above)
Comment (EmiMidnight) [reply namco/io.md:57]: I'm just looking at the plugged in header from the cable, and that cable only has 7 wires going into the connector somehow.

**Pull request merged**